src/main/java/ch/njol/skript/expressions/ExprDrops.java

```diff
@@ -61,28 +64,32 @@ public class ExprDrops extends SimpleExpression<ItemType> {
 
 	@Override
 	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
-		if (!getParser().isCurrentEvent(EntityDeathEvent.class)) {
-			Skript.error("The expression 'drops' can only be used in death events", ErrorQuality.SEMANTIC_ERROR);
+		if (!getParser().isCurrentEvent(EntityDeathEvent.class, BlockDropItemEvent.class)) {
+			Skript.error("The expression 'drops' can only be used in death events and block drop events");
 			return false;
 		}
 		return true;
 	}
 
 	@Override
-	@Nullable
-	protected ItemType[] get(Event e) {
-		if (!(e instanceof EntityDeathEvent))
-			return null;
-
-		return ((EntityDeathEvent) e).getDrops()
-			.stream()
-			.map(ItemType::new)
-			.toArray(ItemType[]::new);
+	protected @Nullable ItemType[] get(Event event) {
+		if (event instanceof EntityDeathEvent entityDeathEvent) {
+			return entityDeathEvent.getDrops()
+				.stream()
+				.map(ItemType::new)
+				.toArray(ItemType[]::new);
+		} else if (event instanceof BlockDropItemEvent blockDropItemEvent) {
+			return blockDropItemEvent.getItems()
+				.stream()
+				.map(Item::getItemStack)
+				.map(ItemType::new)
+				.toArray(ItemType[]::new);
+		}
```

Comment:
```suggestion
		}
		assert false;
```
Optional as there aren't any tests to catch this.